core/src/main/java/org/jdbi/v3/core/statement/ParsedSql.java

```diff
@@ -72,6 +72,25 @@ public String toString() {
             + '}';
     }
 
+    /**
+     * A static factory for {@link ParsedSql} object. It takes two
```

Comment:
typo: objects
"It takes two arguments" is kinda redundant to say

Reply:
Will do.

Reply:
Done.